### ISSUE TITLE
Lumion Pro v10.0

### ISSUE THREAD

**Issue by Hrishikesh-K** [Hrishikesh-K/WinHackerman#184]
The URL of the page showing these comments: https://winhackerman.netlify.app/LumionPro

**Comment by MoYo74** [comment]
How do you download it, is it free?

**Comment by Hrishikesh-K** [comment]
> How do you download it, is it free?
> 

Yes, it's totally free. You can download it using the links provided. It's divided into multiple parts because of the file size limit of Telegram Messenger. For more info about downloading files, please refer to: https://winhackerman.netlify.app/readme#downloading
